src/main/java/graphql/schema/idl/SchemaGeneratorHelper.java

```diff
@@ -1,22 +1,24 @@
 package graphql.schema.idl;
 
-import graphql.Assert;
 import graphql.Internal;
 import graphql.Scalars;
-import graphql.introspection.Introspection;
+import graphql.introspection.Introspection.DirectiveLocation;
 import graphql.language.Argument;
```

Comment:
A future change could make DirectiveLocation be its own top level class but thats really API breaking